app/views/notifier/review_orders.text.erb

```diff
@@ -1,3 +1,3 @@
-<%= text(".body", facility: @facility.to_s, account: @account.to_s) %>
+<%= text(".body", facility: @facility.to_s, accounts: @accounts.html_safe) %>
```

Comment:
What's getting encoded? Is there html inside `@accounts`? That doesn't feel right. Are they `join`ed?